### PR TITLE
API1600  support for Storage system

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[API1600  support for Storage systems]

### Issues Resolved
[]

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/shared_samples/storage_system.rb [hunk 1]
```diff
@@ -21,6 +21,7 @@
 # - API300 for C7000 and Synergy
 # - API500 for C7000 and Synergy (see /examples/api500/storage_system.rb)
 # - API600 for C7000 and Synergy (see /examples/api600/storage_system.rb)
+# - API1600 for C7000 and Synergy (see /examples/api600/storage_system.rb)
```
Comment (chebroluharika) [inline examples/shared_samples/storage_system.rb:24]: please mention for other API versions also

File: lib/oneview-sdk/resource/api1600/synergy/storage_system.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require_relative '../c7000/storage_system'
+
+module OneviewSDK
+  module API1600
+    module Synergy
+      # Storage system resource implementation for API 1600 Synergy
```
Comment (chebroluharika) [inline lib/oneview-sdk/resource/api1600/synergy/storage_system.rb:17]: Remove space between API and 1600

File: spec/unit/resource/api1600/c7000/storage_system_spec.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
```
Comment (chebroluharika) [inline spec/unit/resource/api1600/c7000/storage_system_spec.rb:1]: copyright is not present. Please add it

File: spec/unit/resource/api1600/synergy/storage_system_spec.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
```
Comment (chebroluharika) [inline spec/unit/resource/api1600/synergy/storage_system_spec.rb:1]: same as above

File: spec/unit/resource/api800/synergy/storage_system_spec.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
```
Comment (chebroluharika) [inline spec/unit/resource/api800/synergy/storage_system_spec.rb:1]: please include copyright

File: spec/unit/resource/api800/synergy/storage_system_spec.rb [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+
+RSpec.describe OneviewSDK::API800::Synergy::StorageSystem do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API800::C7000::VolumeTemplate' do
```
Comment (chebroluharika) [inline spec/unit/resource/api800/synergy/storage_system_spec.rb:6]: it should be StorageSystem in place of VolumeTemplate

File: spec/unit/resource/api800/c7000/storage_system_spec.rb [hunk 7]
```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API800::C7000::StorageSystem do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API600::C7000::VolumeTemplate' do
```
Comment (chebroluharika) [inline spec/unit/resource/api800/c7000/storage_system_spec.rb:6]: it should be StorageSystem in place of VolumeTemplate

File: spec/unit/resource/api800/synergy/storage_system_spec.rb (hunk 6 above)
Comment (nabhajit-ray) [reply spec/unit/resource/api800/synergy/storage_system_spec.rb]: done

File: spec/unit/resource/api1600/synergy/storage_system_spec.rb (hunk 4 above)
Comment (nabhajit-ray) [reply spec/unit/resource/api1600/synergy/storage_system_spec.rb:12]: done

File: spec/unit/resource/api1600/c7000/storage_system_spec.rb (hunk 3 above)
Comment (nabhajit-ray) [reply spec/unit/resource/api1600/c7000/storage_system_spec.rb:12]: done

File: lib/oneview-sdk/resource/api1600/synergy/storage_system.rb (hunk 2 above)
Comment (nabhajit-ray) [reply lib/oneview-sdk/resource/api1600/synergy/storage_system.rb]: done

File: examples/shared_samples/storage_system.rb (hunk 1 above)
Comment (nabhajit-ray) [reply examples/shared_samples/storage_system.rb:27]: done

File: spec/unit/resource/api800/synergy/storage_system_spec.rb (hunk 5 above)
Comment (nabhajit-ray) [reply spec/unit/resource/api800/synergy/storage_system_spec.rb:12]: done

File: spec/unit/resource/api800/c7000/storage_system_spec.rb (hunk 7 above)
Comment (nabhajit-ray) [reply spec/unit/resource/api800/c7000/storage_system_spec.rb]: done

**Comment by chebroluharika** [comment]
Please change copyright years in spec files before merging.

**Pull request merged**